tdrs-backend/tdpservice/users/admin.py

```diff
@@ -33,27 +33,29 @@ def clean(self):
             'Developer': 'stt'
         }
 
-        correct_location_type = role_location_type_map.get(group.name)
-        location_based_role = group.name in ('OFA Regional Staff', 'Data Analyst', 'Developer')
+        if group is not None:
```

Comment:
For posterity, can you document/explain in the code why we decided on this change?

Reply:
hopefully that captures it, open to suggestions on improving the clarity. also if there's another place i should document the process change, let me know